<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Admin</class>
 <widget class="QMainWindow" name="Admin">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>954</width>
    <height>373</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Admin</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QGridLayout" name="gridLayout">
    <item row="1" column="0">
     <widget class="QTableView" name="WorkerList"/>
    </item>
    <item row="1" column="1">
     <widget class="QTableView" name="TaskList"/>
    </item>
    <item row="2" column="1">
     <widget class="QGroupBox" name="groupBox_2">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout_2">
       <item>
        <widget class="QPushButton" name="setDoneButton">
         <property name="text">
          <string>Сдать работу</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="addTaskButton">
         <property name="text">
          <string>Добавить задание</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteTaskButton">
         <property name="text">
          <string>Удалить задание</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="2" column="0">
     <widget class="QGroupBox" name="groupBox">
      <property name="title">
       <string/>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QPushButton" name="addEmployerButton">
         <property name="text">
          <string>Добавить работника</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="deleteEmployerButton">
         <property name="text">
          <string>Удалить работника</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item row="0" column="0">
     <widget class="QLabel" name="workerListTitle">
      <property name="text">
       <string>Список работников</string>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QLabel" name="taskListTitle">
      <property name="text">
       <string>Список текущих заданий</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>954</width>
     <height>20</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>Файл</string>
    </property>
    <addaction name="addEmployerAction"/>
    <addaction name="addTaskAction"/>
    <addaction name="reloadWindow"/>
    <addaction name="save"/>
    <addaction name="exit"/>
   </widget>
   <widget class="QMenu" name="menu_2">
    <property name="title">
     <string>Справка</string>
    </property>
    <addaction name="about"/>
   </widget>
   <addaction name="menu"/>
   <addaction name="menu_2"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="addEmployerAction">
   <property name="text">
    <string>Добавить работника</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+W</string>
   </property>
  </action>
  <action name="addTaskAction">
   <property name="text">
    <string>Добавить задание</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+T</string>
   </property>
  </action>
  <action name="reloadWindow">
   <property name="text">
    <string>Обновить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+U</string>
   </property>
  </action>
  <action name="save">
   <property name="text">
    <string>Сохранить данные</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+S</string>
   </property>
  </action>
  <action name="about">
   <property name="text">
    <string>О программе &quot;WorkPlace&quot;</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+A</string>
   </property>
  </action>
  <action name="exit">
   <property name="text">
    <string>Выйти из программы</string>
   </property>
   <property name="shortcut">
    <string>Ctrl+E</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>addEmployerButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>addWorker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>236</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addEmployerAction</sender>
   <signal>triggered()</signal>
   <receiver>Admin</receiver>
   <slot>addWorker()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addTaskButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>addTask()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>698</x>
     <y>416</y>
    </hint>
    <hint type="destinationlabel">
     <x>467</x>
     <y>248</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>save</sender>
   <signal>triggered()</signal>
   <receiver>Admin</receiver>
   <slot>saveWorkPlaceToFile()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>reloadWindow</sender>
   <signal>triggered()</signal>
   <receiver>Admin</receiver>
   <slot>reloadWindow()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteEmployerButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>deleteEmployers()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>deleteTaskButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>deleteTasks()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>319</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>WorkerList</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>Admin</receiver>
   <slot>editWorker(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>241</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TaskList</sender>
   <signal>activated(QModelIndex)</signal>
   <receiver>Admin</receiver>
   <slot>editTask(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>147</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>exit</sender>
   <signal>triggered()</signal>
   <receiver>Admin</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>setDoneButton</sender>
   <signal>clicked()</signal>
   <receiver>Admin</receiver>
   <slot>setTaskDone()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>265</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>TaskList</sender>
   <signal>clicked(QModelIndex)</signal>
   <receiver>Admin</receiver>
   <slot>saveIndex(QModelIndex)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>712</x>
     <y>133</y>
    </hint>
    <hint type="destinationlabel">
     <x>476</x>
     <y>186</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>addWorker()</slot>
  <slot>addTask()</slot>
  <slot>saveWorkPlaceToFile()</slot>
  <slot>reloadWindow()</slot>
  <slot>deleteEmployers()</slot>
  <slot>deleteTasks()</slot>
  <slot>editWorker(QModelIndex)</slot>
  <slot>editTask(QModelIndex)</slot>
  <slot>saveIndex(QModelIndex)</slot>
  <slot>setTaskDone()</slot>
 </slots>
</ui>
